frontend/src/routes/list/+page.ts

```diff
@@ -0,0 +1,27 @@
+import {get} from "../../lib/openapi";
+import * as https from "https";
+import * as fs from "fs";
+import * as path from "path";
+
+/** @type {import('./$types').PageLoad} */
+export async function load() {
```

Comment:
This function should call the Caddy container to get data from the api.

The host url should be `https://localhost/...`, but since we're inside a container, I'm trying to replace it with `https://caddy/...` , but it doesn't work.